plugins/varnish/varnish4_multigraph

```diff
@@ -0,0 +1,453 @@
+    print('sess_conn.type DERIVE')
+    print('cache_miss.label Cache misses')
+    print('cache_miss.min 0')
+    print('cache_miss.draw STACK')
+    print('cache_miss.colour FF0000')
+    print('cache_miss.type DERIVE')
+    print('backend_conn.label Backend conn. success')
+    print('backend_conn.min 0')
+    print('backend_conn.colour 995599')
+    print('backend_conn.type DERIVE')
+    print('s_pass.label Total pass-ed requests seen')
+    print('s_pass.min 0')
+    print('s_pass.colour 785d0d')
+    print('s_pass.type DERIVE')
+    print('backend_unhealthy.label Backend conn. not attempted')
+    print('backend_unhealthy.min 0')
+    print('backend_unhealthy.colour FF55FF')
+    print('backend_unhealthy.type DERIVE')
+    print('cache_hitpass.label Cache hits for pass')
+    print('cache_hitpass.min 0')
+    print('cache_hitpass.draw STACK')
+    print('cache_hitpass.colour FFFF00')
+    print(
+        'cache_hitpass.info Hitpass are cached passes: An entry in the '
+        'cache instructing Varnish to pass. Typically achieved after a '
+        'pass in vcl_fetch.')
+    print('cache_hitpass.type DERIVE')
+    print('cache_hit.label Cache hits')
+    print('cache_hit.min 0')
+    print('cache_hit.draw AREA')
+    print('cache_hit.colour 00FF00')
+    print('cache_hit.type DERIVE')
+
+    # threads
+    print('multigraph varnish_threads')
+    print('graph_category webserver')
+    print('graph_title Thread status')
+    print('threads_created.label Threads created')
+    print('threads_created.min 0')
+    print('threads_created.type DERIVE')
+    print('threads_failed.label Thread creation failed')
+    print('threads_failed.min 0')
+    print('threads_failed.warning :1')
+    print('threads_failed.type DERIVE')
+    print('threads_destroyed.label Threads destroyed')
+    print('threads_destroyed.min 0')
+    print('threads_destroyed.warning :1')
+    print('threads_destroyed.type DERIVE')
+    print('threads_limited.label Threads hit max')
+    print('threads_limited.min 0')
+    print('threads_limited.type DERIVE')
+    print('threads.label Total number of threads')
+    print('threads.min 0')
+    print('threads.warning 1:')
+    print('threads.type GAUGE')
+
+    # transfer_rates
+    print('multigraph varnish_transfer_rates')
+    print('graph_category webserver')
+    print('graph_title Transfer rates')
+    print('graph_order s_resp_bodybytes s_resp_hdrbytes')
+    print('graph_vlabel bit/s')
+    print('graph_args -l 0')
+    print('s_resp_bodybytes.label Body traffic')
+    print('s_resp_bodybytes.min 0')
+    print('s_resp_bodybytes.draw AREA')
+    print('s_resp_bodybytes.cdef s_resp_bodybytes,8,*')
+    print('s_resp_bodybytes.type DERIVE')
+    print('s_resp_hdrbytes.label Header traffic')
+    print('s_resp_hdrbytes.min 0')
+    print('s_resp_hdrbytes.draw STACK')
+    print('s_resp_hdrbytes.cdef s_resp_hdrbytes,8,*')
+    print(
+        's_resp_hdrbytes.info HTTP Header traffic. TCP/IP overhead is not '
```

Comment:
Feel free to use a line limit of 99 characters. But shorter lines are of course also allowed :)